custom_components/hacs/.translations/no.json

```
@@ -19,7 +19,7 @@
             "auth": "Personlig tilgangstoken er ikke korrekt."
         },
         "abort": {
-            "single_instance_allowed": "Bare en enkelt konfigurasjon av HACS er tillatt."
+            "single_instance_allowed": "Bare en enkel konfigurasjon av HACS er tillatt."
```

```suggestion
            "single_instance_allowed": "Bare en konfigurasjon av HACS er tillatt."
```

I think you can just remove the "enkel" from this string